examples/with-flow/pages/about.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import React from 'react'
```

Comment:
With React 17 (and Next.js), we don't need to include this import statement. Should this be handled with Flow instead?

Reply:
I will try to handle import stament with Flow.
Thank you for your review !